gap/fp/word.gi

```diff
@@ -0,0 +1,14 @@
+InstallMethod(WordToString, "for a string and a list",
+[IsString, IsList],
+function(alphabet, word)
+  local str, i, j;
+   str := "";
```

Comment:
Better to assign `str` to a value only after checking that `alphabet` and `word` are compatible.